src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.tsx

```diff
@@ -165,9 +183,13 @@ export const ContactFlowColumn: React.FC<Props> = ({
                 accountListId={accountListId}
                 contact={contact}
                 appealStatus={appealStatus}
-                contactStatus={contact.status}
                 onContactSelected={onContactSelected}
                 columnWidth={cardContentRef.current?.offsetWidth}
+                excludedContacts={
+                  isExcludedContact
+                    ? excludedContacts?.appeal?.excludedAppealContacts ?? []
+                    : []
```

Comment:
If you implement the suggestion above.

```suggestion
                  excludedContacts?.appeal?.excludedAppealContacts ?? []
```